app/components/admin/editions/history_mode_form_controls.erb

```diff
@@ -20,6 +20,10 @@
           name: "edition[government_id]",
           label: "Change the associated government:",
           id: "edition_government_id",
+          data_attributes: {
+            ga4_document_type: "#{action_name}-#{controller_name}",
+            ga4_section: "Change the associated government",
```

Comment:
Could we take the label and use that for populating the `section` of the event? If it needed to be different for a specific component, we could add logic that would give a specified document type on the select component being changed precedence.